### ISSUE TITLE
conceal: support composed character as cchar

### ISSUE THREAD

**Issue by aloispichler** [neovim/neovim#17674]
### Neovim version (nvim -v)

0.6.1

### Vim (not Nvim) behaves the same?

yes, vim 8.2.

### Operating system/version

ArchLinux

### Terminal name/version

xfce4-terminal

### $TERM environment variable

xterm-256

### Installation

AUR

### How to reproduce the issue

This
`:syntax match Entity "tildeP" conceal cchar=p̃`
does not work, probably because p̃ is composed from multiple bytes.
The character displayed is `p` instead of `p̃`.
Same with other characters.

Would be nice to use conceal all utf8 characters.

The setting is
`fileencoding=utf-8`

### Expected behavior

The concealed character should be: 
`p̃`

### Actual behavior

The output is:
`p`

**Comment by theHamsta** [comment]
Should be supported now via https://github.com/neovim/neovim/pull/17776

**Comment by theHamsta** [comment]
Nope, I guess that's still not possible for single letters just for whole nodes.

**Comment by ggustafsson** [comment]
This is kind of related to the older issue #15353, differents needs but both could be resolved by allowing multiple chars (or at minimum extended chars).

**Comment by zeertzjq** [comment]
For extmarks this is supported with #26249.
